templates/prelogin/_sections/pricing/05_data_security.html

```diff
@@ -11,114 +12,36 @@
-                        <div class="pricing-info">
-                            <p><i class="fa fa-minus"></i></p>
-                        </div>
-                    </div>
-                    <div class="col-xs-2 pricing-col-pro">
-                        <div class="pricing-info">
-                            <p><i class="fa fa-check"></i></p>
-                        </div>
-                    </div>
-                    <div class="col-xs-2 pricing-col-advanced">
-                        <div class="pricing-info">
-                            <p><i class="fa fa-check"></i></p>
-                        </div>
-                    </div>
-                    <div class="col-xs-2 pricing-col-enterprise">
-                        <div class="pricing-info">
-                            <p><i class="fa fa-check"></i></p>
-                        </div>
-                    </div>
+                    {% trans "Deidentified data" as feature_name %}
+                    {% pricing_row_contents feature_name "pro" %}
                 </div>
                 <div class="row pricing-row pricing-row-main pricing-row-double
                             pricing-row-triple-xs">
-                    <div class="col-xs-2">
-                        <h4>
-                            {% blocktrans %}
-                                HIPAA compliance assurance
-                            {% endblocktrans %}
-                        </h4>
-                    </div>
-                    {# x x x * * #}
-                    <div class="col-xs-2 pricing-col-community">
-                        <div class="pricing-info">
-                            <p><i class="fa fa-minus"></i></p>
-                        </div>
-                    </div>
-                    <div class="col-xs-2 pricing-col-standard">
-                        <div class="pricing-info">
-                            <p><i class="fa fa-minus"></i></p>
-                        </div>
-                    </div>
-                    <div class="col-xs-2 pricing-col-pro">
-                        <div class="pricing-info">
-                            <p><i class="fa fa-minus"></i></p>
-                        </div>
-                    </div>
-                    <div class="col-xs-2 pricing-col-advanced">
-                        <div class="pricing-info">
-                            <p><i class="fa fa-check"></i></p>
-                        </div>
-                    </div>
-                    <div class="col-xs-2 pricing-col-enterprise">
-                        <div class="pricing-info">
-                            <p><i class="fa fa-check"></i></p>
-                        </div>
-                    </div>
+                    {% trans "Security Policy Control and Enforcement" as feature_name %}
+                    {% pricing_row_contents feature_name "advanced" %}
+                </div>
+                <div class="row pricing-row pricing-row-main pricing-row-double
```

Comment:
:+1: do you have a list of screen sizes you recommend checking for future changes? would be good to have that documented since I definitely didn't do it (just made it work on my screen and used the same classes as sections that had similar amounts of text)